html/index.html

```diff
@@ -0,0 +1,480 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Urbanist:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+    </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+
+    <link rel="stylesheet" href="../css/index.css" />
+    <title>Document</title>
+
+    <style>
+        *{
+            font-family: Poppins;
+        }
+    </style>
+  </head>
+  <body>
+
+    <div class="wrapper">
+        <div class="top-container">
+
+
+            <div class="top-bar-connection">
+    
+                <div class="time-connection">
+    
+                    <img src="../assets/Bars/Status Bars/↳ Time.png" alt="">
+    
+                </div>
+    
+                <div class="b-w-b-connection">
+    
+                    <img src="../assets/Bars/Status Bars/Cellular Connection.png" alt="">
+                    <img src="../assets/Bars/Status Bars/Wifi.png" alt="">
+                    <img src="../assets/Bars/Status Bars/Battery.png" alt="">
+    
+                </div>
+    
+            </div>
+    
+            <div class="header">
+    
+    
+            
+    
+                <div class="left-text">My Logo</div>
+    
+                <div class="right-hamburger">
+    
+                    <img src="../assets/Union.png" alt="">
+    
+                </div>
+    
+            </div>
+    
+            <div class="top-body-background">
+                <div class="top-body">
+    
+                    <h2>I'm Toyyib, Glad to see you!</h2>
```

Comment:
- [ ] Kindly make sure your main title is a `h1` tag instead of an `h2` this ensures google (and other search engines) knows about a specific topic. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.